src/screens/OtherProfile/index.js

```diff
@@ -186,39 +108,6 @@ const OtherProfile = () => {
     setAnonProfile(anonProfileResult);
   };
 
-  const sentAnonDM = async () => {
-    try {
-      setLoadingSendDM(true);
-      const {
-        anon_user_info_emoji_name,
-        anon_user_info_emoji_code,
-        anon_user_info_color_name,
-        anon_user_info_color_code
-      } = anonProfile ?? {};
-
-      const anonDMParams = {
-        user_id: dataMain.user_id,
-        message: dmChat,
-        anon_user_info_emoji_name,
-        anon_user_info_emoji_code,
-        anon_user_info_color_name,
-        anon_user_info_color_code
-      };
-      const response = await sendAnonymousDMOtherProfile(anonDMParams);
-      await saveChatFromOtherProfile(response, 'sent', true, ANON_PM);
-      setDMChat('');
-    } catch (e) {
-      if (e?.response?.data?.status === 'Channel is blocked') {
-        const response = e?.response?.data?.data;
-        await savePendingChatFromOtherProfile(response, true);
-        setDMChat('');
-        return;
-      }
-    } finally {
-      setLoadingSendDM(false);
-    }
-  };
-
   const showSignedMessageDisableToast = () => {
     if (!isSignedMessageEnabled) {
       ToastMessage.show({
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [108-119]

The function `sentAnonDM` is removed. This function was responsible for sending anonymous direct messages. Ensure that this functionality is now managed elsewhere in your application.
```diff
- const sentAnonDM = async () => {
-   // implementation...
- };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ok